sdk/maps/Azure.Maps.Route/samples/RouteDirectionsSamples.md

```diff
@@ -0,0 +1,196 @@
+# Route Directions Samples
+
+To use these samples, you'll first need to set up resources. See [getting started](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/maps/Azure.Maps.Route#getting-started) for details.
```

Comment:
I did not do a deep review of the text content of each sample.   I'd suggest having someone review for grammar and readability.